src/collections/_documentation/platforms/cocoa/enableAutoSessionTracking/objc.md

```diff
@@ -0,0 +1,6 @@
+```objc
```

Comment:
is this the right one?  I think the other docs have `objective-c`

Reply:
`objective-c` is wrong I tested it and everywhere else it's `objc`